src/utils/auth.ts

```diff
@@ -32,6 +39,23 @@ export function isAccessTokenExpiring(token: string) {
   return timeToLive <= 60 * 60 // 1 hour
 }
 
+/**
+ * Checks if the Refresh Token is expired or expiring in 30 secs. A default Refresh Token
+ * lasts 30 days.  Once the Refresh Token expires, the user must re-authenticate (provide
+ * credentials in a browser to obtain an authorisation code). 30 seconds is enough time
+ * to make a request for a final Access Token.
+ * @param {string} token- token string that will be evaluated
+ */
+export function isRefreshTokenExpired(token: string): boolean {
```

Comment:
it makes sense to call this function `isRefreshTokenExpiring` to correspond `isAccessTokenExpiring`

Reply:
done